### PR TITLE
Update script-greetings.rpy

### PR DESCRIPTION
**Pull request by RageofRagaki**
Adds explainations on why work was so amazing

### CODE REVIEW

File: Monika After Story/game/script-greetings.rpy [hunk 1]
```diff
@@ -3071,6 +3071,23 @@ label greeting_back_from_work:
...
+                            m 2eua "I knew you could do it."
+
+                        "I got a lot done!":
+                            m 3hub "That's great [mas_get_player_nickname()]!"
+                            m 5eua "You're such a hard worker, keep it up and you might move up soon!"
+
+                        "It was just an amazing day.":
+                            m 1hua "That's good to hear."
+                            m 4eua "Just a nice and easy day at work is refreshing in it's own right."
+                            m 3tsb "And what better way to wrap up an amazing day with your amazing girlfriend?"
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:3090]:
```suggestion
                m 1sub "That's {i}amazing{/i}, [player]!"
                m 1hub "I'm really happy that you had such a great day!"
                m 2sua "What made it such an amazing day?{nw}"
		        menu:
                    m "What made it such an amazing day?{fast}"

                    "I moved up!":
                        m 6suo "Wow! Congratulations [mas_get_player_nickname()]!"
                        m 1hua "I'm so proud of you!"
                        m 2eua "I knew you could do it."

                    "I got a lot done!":
                        m 3hub "That's great [mas_get_player_nickname()]!"
                        m 5eua "You're such a hard worker, keep it up and you might move up soon!"

                    "It was just an amazing day.":
                        m 1hua "That's good to hear."
                        m 4eua "Just a nice and easy day at work is refreshing in it's own right."
                        m 3tsb "And what better way to wrap up an amazing day with your amazing girlfriend?"
```

File: Monika After Story/game/script-greetings.rpy [hunk 2]
```diff
@@ -3071,6 +3071,23 @@ label greeting_back_from_work:
             "Amazing.":
                 m 1sub "That's {i}amazing{/i}, [player]!"
                 m 1hub "I'm really happy that you had such a great day!"
+                m 2sua "What made it such an amazing day?{nw}"
+		        menu:
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:3075]:
```suggestion
		          menu:
```

**Comment by jmwall24** [comment]
Note: this PR is failing travis because the sideways smug eyes have not yet been added as valid in the sprite checker

**Comment by RageofRagaki** [comment]
Closed because john wanted to be a dumbass and try to fix something that was working fine.

**Comment by jmwall24** [comment]
If by working fine you meant was both repetitive and conflicting, sure.

**Comment by kkrosie123** [comment]
Please don't be disrespectful to the devs.

**Pull request closed without merging**